docs/watchlists-feeds-reports.rst

```diff
@@ -0,0 +1,244 @@
+against the "Enterprise EDR" entry.
+
+All examples here assume that a Carbon Black Cloud SDK connection has been set up, such as with the following code:
+
+::
+
+    >>> from cbc_sdk import CBCloudAPI
+    >>> api = CBCloudAPI(profile='sample')
+
+Setting Up a Basic Custom Watchlist
+-----------------------------------
+Creating a custom watchlist that can watch and/or generate alerts requires three steps:
+
+1. Create a report including one or more Indicators of Compromise (IOCs).
+2. Add that report to a watchlist.
+3. Enable alerting on the watchlist.
+
+Creating a Report
++++++++++++++++++
+We create the report, adding one or more IOCs to it:
+
+::
+
+    >>> from cbc_sdk.enterprise_edr import Report
+    >>> report = Report(api, None, {"description": "Unsigned processes impersonating browsers",
+    ...                             "iocs_v2": [{
+    ...                                 "id": "unsigned-chrome",
+    ...                                 "match_type": "query",
+    ...                                 "values": [("process_name:chrome.exe NOT process_publisher_state:FILE_SIGNATURE_STATE_SIGNED")]
+    ...                                 }],
+    ...                             "link": None,
+    ...                             "severity": 5,
+    ...                             "tags": ["compliance", "unsigned browsers"],
+    ...                             "timestamp": 1601326338,
+    ...                             "title": "Unsigned Browsers"})
+    >>> report.save_watchlist()
+
+Reports should always be given a ``title`` that's sufficiently unique within your organization, so as to minimize
+the chances of confusing two or more Reports with each other.  Carbon Black Cloud will generate unique ``id`` values
+for each report, but does not enforce any uniqueness constraint on the ``title`` of reports.
+
+Alternatively, we can update an existing report, adding more IOCs and/or replacing existing ones.  Note that we have
+to find a report associated with a watchlist by looking in its ``reports`` collection.
+
+Adding the Report to a Watchlist
+++++++++++++++++++++++++++++++++
+Now, add the new Report to a new Watchlist:
+
+::
+
+    >>> from cbc_sdk.enterprise_edr import Watchlist
+    >>> watchlist = Watchlist(api, None, {"alerts_enabled": False,
+    ...                                   "classifier": None,
+    ...                                   "description": "We are on the lookout for any signs of suspicious applications running on our endpoints",
+    ...                                   "name": "Suspicious Applications",
+    ...                                   "report_ids": [report.id],
+    ...                                   "tags_enabled": True})
+    >>> watchlist.save()
+
+**Note:** ``classifier`` should always be ``None``, unless the new watchlist is being subscribed to a feed.
+
+When either ``alerts_enabled`` or ``tags_enabled`` are ``True``, that Watchlist will create data you can act on -
```

Comment:
This seems out of place given the next step is `Enabling Alerting on a Watchlist`. Can we move this into the next step as part of the summary?

Reply:
Moving it.